tests/integration/modules/mac_service.py

```diff
@@ -70,17 +63,18 @@ def test_launchctl(self):
         '''
         # Expected Functionality
         self.assertTrue(
-            self.run_function('service.launchctl', ['error', 'bootstrap', 64]))
+            self.run_function('service.launchctl',
+                              ['error', 'bootstrap', '64']))
         self.assertEqual(
             self.run_function('service.launchctl',
-                              ['error', 'bootstrap', 64],
+                              ['error', 'bootstrap', '64'],
```

Comment:
Could you clarify why this needs to be a string? If strings are required parameters for this function, it seems like we should be handling them in the function itself or at the very least documenting that fact.